src/pages/GapsPage.tsx

```diff
@@ -43,11 +43,37 @@ const Cell = styled.div`
 const TitleCell = styled(Cell)`
   font-weight: bold;
 `
+const TitleCellPrecentage = styled(TitleCell)`
+  width: 240px;
+`
+function displayGapsPercentage(gapsPrecentage: number) {
```

Comment:
can become a small component
```suggestion
function GapsPercentage({gapsPrecentage: number}) {
```